cypress/integration/createNewUser.js

```diff
@@ -0,0 +1,32 @@
+/// <reference types="cypress" />
+import { faker } from "@faker-js/faker";
+
+describe("login for itera ", () => {
+  before(function () {
+    cy.URL();
+    cy.login();
+  });
+  beforeEach(function () {
+    Cypress.Cookies.preserveOnce("ASP.NET_SessionId");
+  });
+  it("should add new user and verify added data is exist in list", () => {
+    const name = faker.name.fullName();
+    const company = faker.company.bs();
+    const address = faker.address.streetAddress(true);
+    const phone = faker.phone.number("984######");
+    const email = faker.internet.email();
+    const city = faker.address.city();
+    const data = [name, company, address, city, phone, email];
+    cy.get(".btn-primary").click();
```

Comment:
Add a line between two different actions: between variable declaration and commands